locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -394,14 +396,14 @@ msgstr "Hap automatikisht Përgjigjet Direkte "
 #. Used to show the numerical rating, when the user hovers over a visual star-rating. Sorry, this is hard to translate.
 msgctxt "homepage ATB social proof"
 msgid "Average rating: %s out of 5"
-msgstr ""
+msgstr "Vlerësimi mesatar %s nga 5"
 
 msgid "Avoid eavesdropping and enhance wi-fi security."
 msgstr ""
 
 msgctxt "homepage onboarding"
 msgid "Back to search"
-msgstr ""
+msgstr "Evito përgjimin e paketave të rrjetit duke përmirësuar sigurinë e Wi-Fisë"
```

Comment:
You added this translation at the wrong place. Here you should add the translation for "Back to search", or just remove this. I added this to the appropriate place a few lines above as a suggestion. Thanks!